.github/workflows/android-release-workflow.yml

```diff
@@ -1,22 +1,21 @@
 on:
   pull_request:
     branches:
-      - release
+      - main
```

Comment:
what should we do with hotfix branches that would fix bugs on release build?
the branch would be feature/PWN-XXX to -> release/vX.X.X, how can we get the release build there?